docs/content/latest/explore/ysql-language-features/tablespaces.md

```diff
@@ -0,0 +1,162 @@
+---
+title: Tablespaces
+linkTitle: Tablespaces
+description: Tablespaces in YSQL
+headcontent: Tablespaces in YSQL
+image: /images/section_icons/secure/create-roles.png
+menu:
+  latest:
+    identifier: explore-ysql-language-features-tablespaces
+    parent: explore-ysql-language-features
+    weight: 300
+isTocNested: true
+showAsideToc: true
+---
+
+This document provides an overview of tablespaces in YSQL and demonstrates how they can be used to specify data placement in the cloud. 
+
+This document guides you through the process of creating the following:
+
+- Multi-zone and single-zone tablespaces.
+- Tables that use multi-zone and single-zone tablespaces.
+- Indexes for tables that use multi-zone and single-zone tablespaces.
+
+## Overview
+
+YSQL tablespaces are entities that specify how data associated with them should be replicated and distributed across cloud, regions, and zones.
+
+In PostgreSQL, tablespaces are used for specifying a location on disk with options to control data access. That is, tablespaces group tables and indexes based on how you intend to store and access the data. For instance, you can choose to place heavily accessed smaller tables and indexes in SSD or “fast” storage compared to other tables.
+
+For YugabyteDB clusters, however, location does not pertain to disk locations. For a cloud-native distributed database, location pertains to the cloud, region, and zone where the data is supposed to be. Therefore, although YSQL tablespaces are built on PostgreSQL tablespaces that allow you to specify placement of data at a table level, this placement information in YSQL defines the number of replicas for a table and index, as well as how they can be distributed across a set of cloud, regions, and zones. 
+
+Note that you cannot use a tablespace outside of the cluster in which it is defined.
+
+## Defining a Tablespace
+
+You can define a tablespace using the following syntax:
+
+```sql
+CREATE TABLESPACE tablespace_name 
```

Comment:
Is it possible to do some form of syntax highlighting or something to differentiate the words here that are actual syntax vs what needs to be provided by user?

Reply:
Unfortunately, it's not possible to do any text formatting withing a code block (only colors, but that's automatic depending on the "language" selection).
This notation is very typical (used in postgresql.org and most online tutorials), so the users shouldn't have problems interpreting it, especially considering that we are pointing out the fact that this is syntax.